canonical-web-and-design/buy.ubuntu.com
# Update cart checkout

## Done

- Implemented Vanilla, and addressed the comments made in the [recommendations doc](https://docs.google.com/document/d/1PI12dk533l4Q3_vW-rpe7bcUpdn9ibp7BCHsxKxSQC4/edit#)

## QA
- Visit https://canonical-staging.myshopify.com/cart
- See that the empty cart screen looks "Vanilla-ised"
- In the top nav, click "Desktops"
- Under the "Desktop Essential" product, click "Add to cart"
- In the nav, click "Desktops"
- Under the "Desktop Standard" product, click "Add to cart"
- See that the cart page looks "Vanilla-ised"
- See that the comments regarding the cart page on page 2 of the [recommendations doc](https://docs.google.com/document/d/1PI12dk533l4Q3_vW-rpe7bcUpdn9ibp7BCHsxKxSQC4/edit#) have been addressed

Fixes https://github.com/canonical-web-and-design/buy.ubuntu.com/issues/24
